PriceCalculatorKata/Product.cs

```diff
@@ -41,4 +46,35 @@ public float Price
             _price = value;
         }
     }
+
+    public List<Expense> Expenses;
+
+    public void AddExpense(string description, float amount, Constants.ExpenseType expenseType)
+    {
+        var expenseToAdd = new Expense
+        {
+            Name = description,
+            Cost = amount,
+            ExpenseType = expenseType
+        };
+        Expenses.Add(expenseToAdd);
+    }
+
+    public bool HasExpenses()
+    {
+        return Expenses.Any();
+    }
+
+    public void PrintExpenses()
```

Comment:
is it the responsibility of this class?

Reply:
> is it the responsibility of this class?

I thought about moving it to the PriceCalculator class, but thought this would be simpler. Because If I wanted to print the expenses in PriceCalculator, I'd still need a method in the Product class that returns the list of expenses, and then print them.